pkgs/development/libraries/libpanel/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, fetchurl
+, meson
+, ninja
+, pkg-config
+, vala
+, gobject-introspection
+, gi-docgen
+, glib
+, libadwaita
+, gtk4
+, gnome
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libpanel";
+  version = "1.0.1";
+
+  outputs = [ "out" "dev" "devdoc" ];
+  outputBin = "dev";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/libpanel/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "hBtqtx6wcv1lIAI+H3Gqx/8lDGbq37sXyVXaa/QeIwY=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    vala
+    gobject-introspection
+    gi-docgen
+  ];
+
+  buildInputs = [
+    glib
+    libadwaita
+    gtk4
+  ];
```

Comment:
can you please sort these list entries alphabetically?

Reply:
I prefer sorting them in the order they are listed in the build definition since that makes it easier to compare side by side. Though in this case, that was also alphabetic.